src/game/client/components/envelope_state.cpp

```diff
@@ -35,38 +35,27 @@ void CEnvelopeState::EnvelopeEval(int TimeOffsetMillis, int Env, ColorRGBA &Resu
 	if(m_pEnvelopePoints->NumPoints() == 0)
 		return;
 
-	// online rendering
-	if(m_OnlineOnly)
+	// offline rendering (like menu background) relies on local time
+	nanoseconds Time = nanoseconds::zero();
```

Comment:
```suggestion
	nanoseconds Time;
```
Variable gets unconditionally assigned to below.